client/src/Components/Spectrogram/SpectrogramPage.jsx

```diff
@@ -238,24 +242,62 @@ class SpectrogramPage extends Component {
     return { ...newState };
   }
 
+  getActions = () => {
+    return (
+      <div>
+        <Button
+          onClick={() => {
+            alert('Awaiting implementation for annotation ' + test);
+          }}
+        >
+          <FontAwesomeIcon icon={faPencil} />
```

Comment:
Same thing as the other icon
https://github.com/tailwindlabs/heroicons#react

Reply:
Changed